src/pages/je-recrute-afpr-poei/inscription/index.page.tsx

```diff
@@ -359,7 +360,7 @@ export default function JeRecruteAfprPoeiInscription() {
           						/>
           						<TextArea
           							className={styles.textArea}
-          							label="Commentaires ou autres informations utiles"
+          							label="Commentaires ou autres informations utiles (500 caractères maximum)"
```

Comment:
nitpick: je viens de me penser qu'on pourrait faire un 

```suggestion
          							label=`Commentaires ou autres informations utiles (${MAX_CARACTERES_COMMENTAIRE} caractères maximum)`
```

mais pas gravissime